internal/common/retrystrategy/retry_state.go

```diff
@@ -1,11 +1,17 @@
 package retrystrategy
 
 const (
-	RetryStrategyPendingState   = "PENDING"
-	RetryStrategyCompletedState = "COMPLETED"
-	RetryStrategyErrorState     = "ERROR"
-	RetryStrategyPausedState    = "PAUSED"
-	RetryStrategyUpdatingState  = "UPDATING"
-	RetryStrategyIdleState      = "IDLE"
-	RetryStrategyDeletedState   = "DELETED"
+	RetryStrategyPendingState    = "PENDING"
+	RetryStrategyCompletedState  = "COMPLETED"
+	RetryStrategyErrorState      = "ERROR"
+	RetryStrategyPausedState     = "PAUSED"
+	RetryStrategyUpdatingState   = "UPDATING"
+	RetryStrategyInitiatingState = "INITIATING"
+	RetryStrategyIdleState       = "IDLE"
+	RetryStrategyFailedState     = "FAILED"
+	RetryStrategyActiveState     = "ACTIVE"
+	RetryStrategyDeletedState    = "DELETED"
+
+	RetryStrategyPendingAcceptanceState = "PENDING_ACCEPTANCE"
+	RetryStrategyPendingRecreationState = "PENDING_RECREATION"
```

Comment:
Would it make sense to remove `RetryStrategy` prefix? (Follow-up PR). Feels verbose repeating: `retrystrategy.RetryStrategyXXX`

Reply:
Yes good point, have my doubts of doing it in dev branch as this would modify various resources. Can note this change down.